config/clients/js/template/package.mustache

```diff
@@ -22,20 +22,21 @@
     "axios": "^0.27.2"
   },
   "devDependencies": {
-    "@types/jest": "^28.1.7",
-    "@types/node": "^16.11.49",
-    "@typescript-eslint/eslint-plugin": "^5.33.1",
-    "@typescript-eslint/parser": "^5.33.1",
-    "eslint": "^8.22.0",
-    "jest": "^28.1.3",
+    "@types/jest": "^29.1.0",
+    "@types/node": "^18.7.23",
```

Comment:
Just want to confirm that we are good with bumping the types/node to v18 while the version we build on is v16

Reply:
Normally we stick to LTS versions, but considering v18 is becoming LTS in a few weeks, thought bumping the types would be good.

[LTS Schedule](https://github.com/nodejs/Release)